components/cron-pvc/cron-job/job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cleanup-pvc-pipelinerun
+spec:
+  # Repeat each 5 min
+  schedule: "*/5 * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pvc-cleaner
+          containers:
+          - name: cleanup-pvc-pipelinerun
+            image: docker.io/openshift/origin-cli
```

Comment:
I prefer to avoid using of docker.io due to requests limitation. Official image from Redhat - `registry.redhat.io/openshift4/ose-cli`